spec/spec_helper.rb

```diff
@@ -19,6 +14,13 @@
   end
 end
 
+RSpec::Matchers.define :a_color do |expected_color|
```

Comment:
I'm not sure what the usual is but `a_color` seems like a really weird matcher to me? I'm not sure how these work though so :man_shrugging:

Reply:
Was following the precedent in this repository with custom matchers.